### PR TITLE
Scanning wrong key fix

### PR DESCRIPTION
**Pull request by Kamill90**

### CODE REVIEW

File: src/screens/SendCoinsScreen.tsx [hunk 1]
```diff
@@ -377,7 +377,15 @@ class SendCoinsScreen extends Component<Props, State> {
                 }),
               );
             } catch (error) {
-              Alert.alert(error.message);
+              CreateMessage({
```
Comment (KedziaPawel) [inline src/screens/SendCoinsScreen.tsx:380]: on going back you would reset all the form

Comment (Kamill90) [reply src/screens/SendCoinsScreen.tsx:380]: Why? If someone scanned the wrong QRcode it is better to have the send form filled?

Comment (Kamill90) [reply src/screens/SendCoinsScreen.tsx:380]: No, it behaves like a typical onBack call.

Comment (KedziaPawel) [reply src/screens/SendCoinsScreen.tsx:380]: we can go screen back (isn't this happening now anyway?) but the form should be filled as it was before scanning

Comment (KedziaPawel) [reply src/screens/SendCoinsScreen.tsx:380]: oh ok, so the form is still filled

File: src/consts/models.tsx [hunk 2]
```diff
@@ -353,12 +353,14 @@ export type MainCardStackNavigatorParams = {
...
   [Route.TransactionDetails]: { transaction: EnhancedTransaction };
   [Route.ReceiveCoins]: { id: string };
-  [Route.SendCoins]: {
-    fromSecret?: string;
-    fromAddress?: string;
-    fromWallet?: Wallet;
-    toAddress?: string;
-  };
+  [Route.SendCoins]:
+    | {
```
Comment (KedziaPawel) [inline src/consts/models.tsx:357]: unused `|` in front

File: src/screens/SendCoinsScreen.tsx [hunk 3]
```diff
@@ -58,7 +58,7 @@ class SendCoinsScreen extends Component<Props, State> {
   constructor(props: Props) {
     super(props);
     const { route, wallets } = props;
-    const { toAddress, fromWallet } = route.params;
+    const { toAddress, fromWallet } = route.params!;
```
Comment (KedziaPawel) [inline src/screens/SendCoinsScreen.tsx:61]: this `!` is needed here?

Comment (Kamill90) [reply src/screens/SendCoinsScreen.tsx]: not any more

**Review by KedziaPawel — APPROVED** [review]
tested, working correctly

**Pull request merged**